### ISSUE TITLE
Supporting 5.5 and above

### ISSUE THREAD

**Issue by softwarespot** [brandonwamboldt/utilphp#111]
Maybe this library should support versions of PHP that are still receiving security updates? The reason I say this, is because the web is about moving forward and not living in the past! If you support 5.3+, then you're encouraging developers not to upgrade.

**Comment by hopeseekr** [comment]
Yeah, at this point even I have moved off all of my sites off of PHP 5.4, although I did that by mid-2015 and via a completely unorthodox route: I ditched Zend Engine and Zend Corp., before they were bought out, and now my personal and corporate code is 100% HHVM with a smidgen of HackLang! Woot!

I second the motion that we make a final php-5.3 release and focus on php-5.5+ from here on out. Agreed, @brandonwamboldt?

**Comment by AaronHolbrook** [comment]
:+1:
